lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1313,6 +1313,7 @@ data ApiErrorCode
     | SharedWalletNoSuchCosigner
     | SharedWalletCannotUpdateKey
     | SharedWalletScriptTemplateInvalid
+    | MintedNotSpent
```

Comment:
This does not necessarily have to be part of the public API.

The API structure does not allow a user to specify a mint that is not sent to an explicit address, so under the current conditions, this error should never be thrown. Hence it probably doesn't need to be in the public API.

Saying that, a token which is minted but not spent or burnt is something that could happen with some other call to performSelection, so I've included this error as part of the public API.

Reply:
But I'm happy to go either way. We could also consider sending minted values that don't have an explicit receiving address to a change bundle. The "makeChange" function already handles that case, we'd just have to update "performSelection".